lighthouse-core/audits/screenshot-thumbnails.js

```diff
@@ -128,8 +129,8 @@ class ScreenshotThumbnails extends Audit {
       }
       thumbnails.push({
         timing: Math.round(targetTimestamp - speedline.beginning),
```

Comment:
should we also add `timing` to `final-screenshot.js` just to completely match?

Reply:
yeah good call done